context/src/main/java/io/opentelemetry/context/ContextStorage.java

```diff
@@ -102,4 +102,14 @@ static void addWrapper(Function<? super ContextStorage, ? extends ContextStorage
    */
   @Nullable
   Context current();
+
+  /**
+   * Returns the root {@link Context} which all other {@link Context} are derived from.
+   *
+   * <p>The default implementation returns the root {@link ArrayBasedContext}, but subclasses can
```

Comment:
```suggestion
   * <p>The default implementation returns the root {@code ArrayBasedContext}, but subclasses can
```

ArrayBasedContext isn't public so can't link to it